### PR TITLE
Added support for go1.5

### PR DESCRIPTION
**Pull request by soltysh**
@bparees @mfojtik ptal

### CODE REVIEW

File: hack/common.sh [hunk 1]
```diff
@@ -399,11 +399,19 @@ sti::build::ldflags() {
...
+    if [[ "${GO_VERSION[2]}" =~ ^go1.4.*$ ]]; then
+      ldflags+=(-X "${STI_GO_PACKAGE}/pkg/version.majorFromGit" "${STI_GIT_MAJOR}")
+      ldflags+=(-X "${STI_GO_PACKAGE}/pkg/version.minorFromGit" "${STI_GIT_MINOR}")
+      ldflags+=(-X "${STI_GO_PACKAGE}/pkg/version.versionFromGit" "${STI_GIT_VERSION}")
+      ldflags+=(-X "${STI_GO_PACKAGE}/pkg/version.commitFromGit" "${STI_GIT_COMMIT}")
+    else
+      ldflags+=(-X "${STI_GO_PACKAGE}/pkg/version.majorFromGit=${STI_GIT_MAJOR}")
+      ldflags+=(-X "${STI_GO_PACKAGE}/pkg/version.minorFromGit=${STI_GIT_MINOR}")
+      ldflags+=(-X "${STI_GO_PACKAGE}/pkg/version.versionFromGit=${STI_GIT_VERSION}")
+      ldflags+=(-X "${STI_GO_PACKAGE}/pkg/version.commitFromGit=${STI_GIT_COMMIT}")
```
Comment (bparees) [inline hack/common.sh]: these look identical...what am i missing?

Comment (soltysh) [inline hack/common.sh]: The space for go1.4, and `=` sign for go1.5 :wink:

Comment (bparees) [inline hack/common.sh]: yikes.  thanks.

**Comment by bparees** [comment]
lgtm [merge]

**Comment by soltysh** [comment]
Once this gets in, I'll be adding govet similarly to what we have in [openshift/origin](https://github.com/openshift/origin) already.

**Pull request merged**